crypto-ffi/bindings/js/CoreCrypto.ts

```diff
@@ -58,20 +58,10 @@ export enum Ciphersuite {
  * Configuration object for new conversations
  */
 export interface ConversationConfiguration {
-    /**
-     * List of client IDs with administrative permissions
-     * Note: This is currently unused
-     */
-    admins?: Uint8Array[];
```

Comment:
This is fine to remove as our current authorization callbacks do the same thing basically